test/tableRow.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import TableRow from '../src/client/TableRow';
+import {mockData} from './mockData';
+
+describe('When table is created', () => {
+  const wrapper = shallow(<TableRow 
+    key={mockData.seq}
+    seq={mockData.seq} 
+    status={mockData.status}
+    category={mockData.category}
+    title={mockData.title}
+    owner={mockData.owner}
+    priority={mockData.priority}
+  />);
+  it('TableRow should have six cells', () => {
+    expect(wrapper.find('td').length).toBe(6);
+  });
+
+  it('TableRow should have different className', () => {
+    if(wrapper.find('td').first() % 2 === 0){
+      expect(wrapper.find('tr').hasClass('table__row table__row--even')).toBe(true);
+    }
+    else if(wrapper.find('td').first() % 2 === 1){
```

Comment:
Interesting case here.
As we can see, you'd like to test `two` cases, one for `odd` and one for `even`. So, it might be better to separate them into different test cases.

Besides, there's one problem here.
As we can see, we didn't change the `wrapper` here which means we'll always have the same input.
If we can use the same input to get different output, it might be an issue.
But, if we can't, one of the conditions here might not work which means we might need another test case with different input.